src/site/content/en/blog/airshift-perf-optimization/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "5 ways AirSHIFT made their React app responsive"
+subhead: |
+  A real world case study of React SPA performance optimization.
+date: 2019-10-25
+authors:
+  - uskay
+  - yosuke_furukawa
+  - satoshi_arai
+  - kento_tsuji
+hero: hero.png
+alt: A logo of AirSHIFT
+description: |
+  This case study explains how the AirSHIFT team optimized the responsiveness
+  of their desktop React SPA and satisfied their users.
+tags:
+  - post
+  - fast
+---
+
+Website performance is not just about load time. It is critical to provide a fast and responsive experience to users, especially for productivity desktop apps which people use everyday. The engineering team at [Recruit Technologies](https://recruit-tech.co.jp/) went through a refactoring project to improve one of their Web apps, [AirSHIFT](https://airregi.jp/shift/), for better user input performance. Here's how they did it.
+
+## Slow response, less productivity
+AirSHIFT is a desktop Web application that helps store owners, like restaurants and cafes, to manage the shift work of their staff members. Built with React, the Single Page Application provides rich client features including various grid tables of shift schedules cut by day, week, month and more.
+
+<img class="w-screenshot" src="airshift_visual.png" alt="A visual image of AirSHIFT">
+
+To meet user expectations they have been adding new functionalities to the app but as they continue to add features, they started seeing more feedback around slow performance. The engineering manager of the product, Yosuke Furukawa says "We did a user research and one of the store owners even said she would leave her seat to brew coffee after clicking a button… just to kill time waiting for the shift table to load, which was a shocking fact we faced". 
```

Comment:
```suggestion
As the Recruit Technologies engineering team added new features to the
AirSHIFT app, they started seeing more feedback around slow performance.
The engineering manager of AirSHIFT said, Yosuke Furukawa, said:

<blockquote>
  In a user research study, we were shocked when one of the store owners said she
  would leave her seat to brew coffee after clicking a button, just to kill time waiting
  for the shift table to load.
</blockquote> 
```

@uskay was this quote originally given in another language? If this person provided this quote in English, then it's a problem for me to change what they said. But if this is your translation of a quote that was spoken in another language, then it's OK for me to reword the quote.

Reply:
It was in Japanese so if the context and the narrative remains the same, I feel it's fine and better for you to correct if needed.